pkg/ddevapp/ddevapp_test.go

```diff
@@ -333,6 +333,22 @@ var (
 			Type:                          nodeps.AppTypeSilverstripe,
 			DynamicURI:                    testcommon.URIWithExpect{URI: "/", Expect: "<meta name=\"generator\" content=\"Silverstripe CMS 5.0\">"},
 		},
+		// 18: CakePHP
+		{
+			Name:                          "TestPkgCakePHP",
+			SourceURL:                     "https://github.com/CakeDC/test-cakephp/archive/refs/tags/5.0.1.1.tar.gz",
```

Comment:
We'll move this to[ a repo in ddev](https://github.com/ddev/test-cakephp) that can be maintained, updated, etc. I have given you maintainer privileges on that repo.

Reply:
Thank you, I will push the code to this repo and update devapp_test.go

Reply:
Moved!